CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 0.14.0-rc1 / 2015-05-21
+* [CHANGE] Configuration format changed and switched to YAML.
```

Comment:
The Changelog being the main guide for upgraders, this should have a link to the migration docs / tool.